### PR TITLE
feature/perfil-usuario

### PR DESCRIPTION
**Pull request by Dukes08**

### CODE REVIEW

File: lib/pages/editProfile.dart [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+import 'package:flutter_svg/flutter_svg.dart'; //para uso de iconos svg
+
+class EditProfile extends StatelessWidget {
+  
+  bool isObscurePassword = true;
+  
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: appBar(),
```
Comment (Dukes08) [inline lib/pages/editProfile.dart:13]: Do not create a component in the same file, add it to the components folder and then import it in this file

File: lib/pages/editProfile.dart [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+      backgroundColor: Colors.white,
+      body: Container(
+        padding: EdgeInsets.only(left: 15, top: 20, right: 15),
+        child: GestureDetector(
```
Comment (Dukes08) [inline lib/pages/editProfile.dart:17]: Do not wrap everything with the GestureDetector

File: lib/pages/editProfile.dart [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+        onTap: (){
+          FocusScope.of(context).unfocus();
+        },
+        child: ListView(
```
Comment (Dukes08) [inline lib/pages/editProfile.dart:21]: Remove listView it does not add anything to the code

File: lib/pages/editProfile.dart [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+          children: [
+            Center(
+              child: Stack(
```
Comment (Dukes08) [inline lib/pages/editProfile.dart:24]: Use the widget Stack to the child widgets and use positioned to position them

File: lib/pages/editProfile.dart [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+              ],
+            )
+          ],
+        )
+        ),
+      ),
+    );
+  }
+
+  Widget buildTextField(String labelText, String placeholder, bool isPasswordTextField){
```
Comment (Dukes08) [inline lib/pages/editProfile.dart:111]: Do not create a component in the same file, add it to the components folder and then import it in this file

File: lib/pages/editProfile.dart [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+    return Padding(
+      padding: EdgeInsets.only(bottom: 30, left: 50, right: 50),
+      child: TextField(
+        obscureText: isPasswordTextField ? isObscurePassword : false,
+        decoration: InputDecoration(
+          suffixIcon: isPasswordTextField ?
+          IconButton(icon: Icon(Icons.remove_red_eye, color: Colors.grey),
+          onPressed: () {
+            //setState(() {
```
Comment (Dukes08) [inline lib/pages/editProfile.dart:120]: You can not use setState in a Stateless widget. Search documentation about the difference between Stateful and Stateless Widgets

**Review by Dukes08 — COMMENTED** [review]
Please make the changes as soon as possible.

File: lib/pages/editProfile.dart (hunk 1 above)
Comment (BeaCardozo) [reply lib/pages/editProfile.dart:13]: finished

File: lib/pages/editProfile.dart (hunk 2 above)
Comment (BeaCardozo) [reply lib/pages/editProfile.dart:17]: finished

File: lib/pages/editProfile.dart (hunk 6 above)
Comment (BeaCardozo) [reply lib/pages/editProfile.dart:120]: Not able to fix it properly

File: lib/pages/editProfile.dart (hunk 4 above)
Comment (BeaCardozo) [reply lib/pages/editProfile.dart:24]: finished

**Pull request merged**